apps/frontend/src/config.ts

```diff
@@ -0,0 +1,14 @@
+const config = {
+  baseURL: 'http://localhost:5173',
+  firebaseConfig: {
+    apiKey: (import.meta.env.VITE_API_KEY as string) ?? '',
```

Comment:
Now if it's undefined we hide the bug with empty string. How about create a get function that throws error if something is undefined.